### PR TITLE
Adding OSGi processing to the pom for java modules

### PR DESCRIPTION
**Pull request by cschneider**

### CODE REVIEW

**Comment by JulianFeinauer** [comment]
Thank you... I will have a look when I’m back at the desk

File: plc4j/pom.xml [hunk 1]
```diff
@@ -45,7 +45,37 @@
 
   <build>
     <plugins>
-
+        <plugin>
```
Comment (JulianFeinauer) [inline plc4j/pom.xml]: Hey @cschneider there is another invocation of the maven-bundle-plugin (L:200). Is this okay or hould this be removed?

Comment (JulianFeinauer) [reply plc4j/pom.xml]: And I would also add the same part (with the part from L:200 with maven-bdunle-plugin and jar plugin???) to protocols/pom.xml, right?

Comment (cschneider) [reply plc4j/pom.xml]: Oh .. sorry I missed that it was already present. So yes the other invocation should be removed. Actually to make the diff look cleaner I will merge my changes into the existing config.

Comment (cschneider) [reply plc4j/pom.xml]: It also makes sense to add the same bundling config to protocol. I can update the PR.

Comment (JulianFeinauer) [reply plc4j/pom.xml]: Thank you : )
Then I will check how the bundles look like and if some of them already work (one can hope...).

**Comment by JulianFeinauer** [comment]
Looks good and builds locally, will merge. Congrats @cschneider for your first merged PLC4X PR 🎉 !

**Pull request merged**